### PR TITLE
docs: update GeekyWeb's showcase description

### PR DESCRIPTION
**Pull request by Designatory**
i have updated my website's description (showcase) please merge it in the main repo

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -748,7 +748,7 @@ const Users: User[] = [
   },
   {
     title: 'GeekyWeb',
-    description: 'Global State and Logic Framework for reactive Applications',
+    description: 'Best documentation website for all developers and programmers.',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:751]: Sorry I didn't review your description before🤦‍♂️ However, this description is not very descriptive. Please be more value-neutral, and describe what's included on your site, not just that "it's the best".

**Comment by Josh-Cena** [comment]
@Designatory Please remember to change your description per my review above ^

**Comment by Designatory** [comment]
Okay

Now what about this PR should i close this PR or not?

**Comment by Josh-Cena** [comment]
No, you should edit it within this PR.

**Pull request closed without merging**